lightning/restore/restore.go

```diff
@@ -1296,6 +1299,15 @@ func (rc *RestoreController) checkRequirements(_ context.Context) error {
 	return rc.backend.CheckRequirements()
 }
 
+func (rc *RestoreController) setGlobalVariables(ctx context.Context) error {
```

Comment:
returning error seems unnecessary

Reply:
Because this function is call in `RestoreController.Run`, so the method type must be `func(context.Context) error`